app/src/androidTest/java/org/mozilla/fenix/ui/SearchTest.kt

```diff
@@ -69,13 +63,30 @@ class SearchTest {
 
     @Test
     fun shortcutSearchEngineSettingsTest() {
-        homeScreen { }.dismissOnboarding()
-
         homeScreen {
         }.openSearch {
             scrollToSearchEngineSettings()
             clickSearchEngineSettings()
             verifySearchEngineSettings()
         }
     }
+
+    @Test
+    fun searchWithDefaultEngineTest(){
+        homeScreen {
+        }.openSearch {
+        }.performSearch("mozilla"){
+            verifyNewTabLoaded()
```

Comment:
Do we want to verify that the default engine was used? Perhaps not necessary, but with `shortcutButtonTest` I was doing a verification that DuckDuckGo was being used: https://github.com/mozilla-mobile/fenix/pull/4770/files#diff-a158074d726613013cf9fa9f6f0d8d70R57-R59

Reply:
@sblatz we certainly can, but there's also a number of feature areas where we could theoretically be testing the same thing repeatedly but from different vectors.  In some cases this overlap is ok, but for this feature, I was planning on verifying the search engine selection with the 3dot menu > settings tests.  That way we can also verify switching to more than one other default search engine.

Reply:
I'll remove this test then.